### PR TITLE
Hotfixes for total_results and data in event_results

### PR DESCRIPTION
**Pull request by jasondilworth56**
These two methods work with these small fixes.

### CODE REVIEW

**Review by lowlyocean — APPROVED** [review]

**Comment by jasondilworth56** [comment]
Sorry for the n00b question, but can we/how do we get this merged into master? @jeysonmc @lowlyocean

File: ir_webstats/client.py [hunk 1]
```diff
@@ -200,6 +209,7 @@ def career_stats(self, custid=None):
         """ Gets career stats (top5, top 10, etc.) of driver (custid)."""
         r = self.__req(ct.URL_CAREER_STATS % (custid),
                        cookie=self.last_cookie)
+        #print(r)
```
Comment (frankivo) [inline ir_webstats/client.py]: this should not be here

Comment (jasondilworth56) [reply ir_webstats/client.py]: Agreed. Forgive my newbieness, but how do I go about making that change into this pull request?

Comment (frankivo) [reply ir_webstats/client.py]: You could just pull the code of the branch again. 
Though I doubt the original author will ever merge it. He hasn't responded to any of the pull requests.

https://github.com/a-tal/irace
This project seems more active :)

**Pull request closed without merging**